README.md

```diff
@@ -248,12 +249,7 @@ spec:
 status:
   binding:              # LocalObjectReference, optional
     name:               # string
-  conditions:           # []Condition containing at least one entry for `Ready`
-  - type:               # string
-    status:             # string
-    lastTransitionTime: # Time
-    reason:             # string
-    message:            # string
+  conditions:           # []metav1.Condition containing at least one entry for `Ready`
```

Comment:
I think this can be reverted. The schema is defined in the KEP-1623 which is mentioned in the normative text above. This schema can be considered as informational.

Reply:
```suggestion
  conditions:           # []Condition containing at least one entry for `Ready`
```

Reply:
On second thought, I'm reverting this suggestion. We also use `metav1` to reference a LabelSelector rather than spell out the shape explicitly.